data/helpers.d/utils

```diff
@@ -24,3 +24,19 @@ ynh_get_plain_key() {
         fi
     done
 }
+
+# Curl abstraction to simplify its call
+#
+# Helpfully mostly for curl arguments
+#
+# example: ynh_curl_abstract "value1=$var1&value2=$var2" "/install.php?installButton"
```

Comment:
I suggest to switch $1 and $2, it allow to write `ynh_curl_abstract "/install.php?installButton"` if there are no POST data